### DISCUSSION TITLE
Mongo query native

### DISCUSSION THREAD

**Discussion by loctd98** [dbeaver/dbeaver#23144] [Q&A]
Hello, Can i view native query mongo when i exec sql statement ?
E.g: `SELECT * FROM user LIMIT 10;` -> `db.getCollection("user").find({}).limit(2);`
And seem the limit() fucntion is not working in my query `db.getCollection("user").find({}).limit(2);` ?
Many thanks!

DBeaver Version
24.0.0.202403110838
Driver
mongo-java-driver 4.5.1

**Comment by LonwoLonwo** [comment]
Hello @loctd98 

It is better to create a ticket for our Tech Support Team: https://dbeaver.com/tickets
(Available for PRO versions)

**Reply by loctd98** [reply]
Yes, i'm using pro version. Can i open new issue to report bug `limit()` not work ?

**Reply by loctd98** [reply]
Do you know pro version has support view native query mongo when exec sql ?

**Comment by iraidaa** [comment]
Hello @loctd98,

These discussions on GitHub are about the DBeaver Community, but we can connect to MongoDB only in DBeaver PRO. Could you please create a ticket in the support system on the website https://dbeaver.com/support/? We investigate issues connected with DBeaver PRO products there.
